src/audio/module_adapter/module_adapter_ipc3.c

```diff
@@ -239,7 +239,7 @@ static int module_adapter_ctrl_get_set_data(struct comp_dev *dev, struct sof_ipc
 					    bool set)
 {
 	int ret;
-	struct processing_module *mod = comp_mod(dev);
+	struct processing_module __maybe_unused *mod = comp_mod(dev);
```

Comment:
At first I didn't like this, because it adds less than pretty code to potentially many locations (we need to handle information-, warning- and error-level messages similarly, right?), but then I thought, that if we have an automated way to identify all these locations, then maybe this **is** the best way to handle them. Firstly it's honest and explicit - it helps keep track of such cases and maybe avoid adding variables that can easily be avoided. The same holds for variables, only used in `assert()` calls, right? So, yes, I'd buy this.